sdk/eventgrid/ci.yml

```diff
@@ -1,7 +1,15 @@
 # DO NOT EDIT THIS FILE
 # This file is generated automatically and any changes will be lost.
 
-trigger: none
+trigger:
```

Comment:
Is there any way to avoid the duplication between trigger and the pr section?

Reply:
Unfortunately there isn't - and I don't think I can push it down it to a template either.